### ISSUE TITLE
Bug Report

### ISSUE THREAD

**Issue by king-kt** [concourse/concourse#4091]
version: v5.3.0
operating system:CentOS Linux release 7.6.1810 (Core)

when I was running the pipeline on the web UI, these issues were emerging.

runc run: exit status 1: container_linux.go:345: starting container process caused "process_linux.go

**Comment by marco-m** [comment]
Hello, could you please do an effort when filing bug reports:
- title doesn't mean anything.
- description doesn't respect the template and the error message is cut in half.

Looking at bug reports takes time, if they are written like this we cannot help :-)

This seems to have something to do with missing executable bit, it is not a bug. I suggest to ask on the Concourse Discord or in the forum. https://concourse-ci.org/project.html.

I am closing this ticket. Feel free to open another one following the template if you found a real bug. Thanks.

**Comment by StevenArmstrong** [comment]
@marco-m dude i have seen you give replies like this on a number of tickets now and i bit my tongue initially. Can i ask that you be kind, or we really do run the risk of creating a toxic community that will discourage people from using concourse. It is pretty clear the issue here is that concourse needs a newer kernel version and cannot run on centos so fedora will need to be used or rhel 8 and i believe you know this. So i implore you to value the code of conduct in your replies going forward.

**Comment by marco-m** [comment]
@StevenArmstrong I think there is a misunderstanding here. Tell me what should I write instead on this specific ticket, or if I should it rot as tens of others. Also go look to all my answers here, on discuss and on discord, and tell me again that I am not kind or that I don't spend time to help.

Regarding the " i believe you know this", I chalk this up to the misunderstanding above.

Peace.

**Comment by StevenArmstrong** [comment]
All i would ask is you put yourself in the place of a newbie filling a ticket for the 1st time. They have their ticket closed, they are told to refill one in, they are quite aggressively in tone told all the things they have done wrong, then sent to a forum where they likely won't get an instant reply while they are stuck with a query on a new tool. 
Are they likely to continue using the tool or give up? 
Sometimes it's good to take a step back and put yourself in another persons shoes. I do not doubt your helpfulness to the community but that doesn't absolve you and mean you get a pass for being overly aggressive or short on responses.
You asked for my advice on the handling of the situation, I will instead pose the question on how you would have liked a response in that situation? 
If you are asking my opinion, in that situation, I would advise them to follow the template on future requests if they were not aware, point them to a good example, ask them to update in the format on the current ticket to match this. I would ask for more details to be placed in the ticket with steps to reproduce and then wait for their response. 
I appreciate it can be frustrating when people don't follow templates or standards but patience and kindness are what make open source communities thrive.

**Comment by marco-m** [comment]
@StevenArmstrong I find fascinating the amount of aggressiveness, rudeness and accusations that you are using with me to teach me that I should not be aggressive and unkind. You even accused me of lying on purpose (the "I believe you know this"). I am not interested in going on in a discussion on this tone.

**Comment by vito** [comment]
@StevenArmstrong I think you're right in that we should be wary of how we reply to issues like this. We should always strive to be empathetic even in cases where users are not adhering to our guidelines. That being said, I think you might have misinterpreted @marco-m's comment to have an aggressive tone when he was not intending it to be. I really think that this is just miscommunication - his comments tend to be short and to the point, which can lead to 'reading between the lines' when there's nothing.

@marco-m I didn't find your tone to be aggressive or rude, I just saw it as stern. Definitely not anywhere close to a breach of the code of conduct IMO. If you're feeling burnt out from replying it's probably easier to just rely on our bots to automatically close issues and give a canned reply in these situations. They are a live encoding of our project policy, have friendly messages, and leave very little to (mis-)interpretation.

In this case the `support` label could have been applied, which would trigger our support bot to swoop in and close the issue with a friendly message directing them to the forums. There's also a `needs-more-information` label which will cause the issue to automatically close if the OP doesn't reply soon enough. Further, all issues that stay open with no activity for 60 days will be automatically marked as 'stale' and closed. This policy is documented in our [wiki](https://github.com/concourse/concourse/wiki/How-Issues-are-Managed#staleness).

---

I may be preaching to the choir at this point as it seems the root of this was a misunderstanding, but I want to explain my stance on why we close issues like this anyway.

I believe that we should close issues like this swiftly, and with empathy, but without wasting too much time.

Taking part in an open-source community, even just reporting bugs, is a two-way street. The product is free, and the price you pay is through contributing to it yourself when you need to - whether that's code or just a detailed-enough bug report.

We have a template suggesting 'good form' for bug reports, and instruct people to use the support forums and Discord if they're not sure exactly whether it's a bug. If all of those guidelines are ignored, that can be interpreted as disrespectful to the project maintainers, just as a short reply + close could be interpreted as disrespectful to the user.

To me you have to 'interpret' as little as possible (barring any obvious personal attacks), judge by the actions, and just enforce the project's policy. I understand that sometimes users are in a hurry or just don't care enough and just want to report a bug without thinking about the people on the other side. But I think in that case you can't expect too much from the project maintainers. You aren't entitled to anything just by being a user.

If we bend over backwards to try and get to a resolution for every single one of these issues, yes we may keep them as a user, and we may even gain more users. But if we're not enforcing policies that help the maintainers keep the project going *with* all those users, all these [broken windows](https://en.wikipedia.org/wiki/Broken_windows_theory) will make the community worse and just lead to burnout for the project maintainers.

@king-kt Sorry this turned into a bit of a debate. 🙂 Let me know if there was a place you would have expected to see these guidelines. We could maybe add stuff like this to `CONTRIBUTING.md`, which is currently very developer-focused.

**Comment by king-kt** [comment]
1. Thank you for all who reply to these issues.
2. As StevenArmstrong said, English is not my 1st language. Therefore, I may not be satisfied in the form of expression.
3. I have understood StevenArmstrong technical reply and it's exactly what she said.
4. After I replaced the operating system and the docker-compose version, the problem was resolved.
5. In my point of view, I think it's caused by a support issue for the Linux system version and the docker-compose tool version.
6. I won't report the issues again after I used ubuntu18 and docker-compose 1.24.0 with Concourse 4.2.1.
Thank you for all.
